device/rsvp/rsvpInterface.yaml

```diff
@@ -0,0 +1,79 @@
+components:
+  schemas:
+    Rsvp.Interface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP Interface.
+      type: object
+      required: [ipv4_name,neighbor_ip]
+      properties:
+        ipv4_name: 
+          description: >-
+            The unique name of the IPv4 interface connected to the DUT.
```

Comment:
WE may clarify that this needs to be unique across the entire configuration across multiple port or the name need to be unique for a single port only.

Reply:
Same as above.